chats/apps/api/v1/internal/projects/viewsets.py

```diff
@@ -75,3 +78,18 @@ def put(
                 status.HTTP_400_BAD_REQUEST,
             )
         return Response({"Detail": "Updated"}, status.HTTP_200_OK)
+
+    @action(detail=True, methods=["GET"], permission_classes=[IsAuthenticated, SectorAnyPermission])
```

Comment:
To make this endpoint more generic, turn it into a POST and get its status in the body, something like:
```json
{
    "status": "ONLINE"
}
```
- Rename this method to: `status`
- you must get the body sent from the request, like this:
https://github.com/weni-ai/chats-engine/blob/0c2fb56b31e05bc371c739778f16240730ff2eeb/chats/apps/api/v1/rooms/viewsets.py#L57

Reply:
changes applied.